frontend/src/__tests__/cypress/cypress/e2e/pipelines/PipelineRuns.cy.ts

```diff
@@ -466,6 +514,28 @@ describe('Pipeline runs', () => {
         cy.wait('@disableJob');
       });
 
+      describe('Navigation', () => {
```

Comment:
Are you testing the pipeline navigation breadcrumbs? When on run details you should be able to navigate back to the view page

Reply:
same thing pipeline and version details

Reply:
i raised a discussion [here](https://redhat-internal.slack.com/archives/C069KSM8T9N/p1709840980679379) for the breadcrumb test, because the test id needs to be added everywhere so I am looking for a better solution.

Reply:
gotcha. looks like there will be some extra effort that is out of scope for this PR. let us just make sure to manually test all of the breadcrumbs before we merge and we can circle back to having a the dashboard-breadcrumb with proper tests later on. Do you mind logging a jira for this (i imagine tech debt, but maybe christian wants it logged differently not sure)

Reply:
Sure, I am logging this.